frontend/src/components/pages/GlobalCodeReviewDashboard/FilterByNames/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Presentation from "./Presentation";
+
+export default function FilterByNames({
+  allNames,
+  filterInclude,
+  filterExclude,
+  onChange,
+  searchTerm,
+  handleChangeSearchTerm,
```

Comment:
searchTerm and handleChangeSearchTerm should not be props. They should be state in this component.

Reply:
If you set up the state close to where it needs to be used then the code is DRY and cohesive.

```
export default function FilterByNames({
  allNames,
  filterInclude,
  filterExclude,
  onChange,
}) {

const [searchTerm, setSearchTerm] = useState()  // This is the only new piece of state we need

function handleChangeSearchTerm(...){
...
setSearchTerm(...)
}

```